src/Form.java

```diff
@@ -3,28 +3,19 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
-
```

Comment:
improved format (in Java, we divide code with 1 line, not with 2 lines as - I guess - in the python)